docs/source/type_inference_and_annotations.rst

```diff
@@ -44,28 +44,11 @@ type:
 
    x: Union[int, str] = 1.1  # Error!
 
-The variable annotation syntax is available starting from Python 3.6.
-In earlier Python versions, you can use a special comment after an
-assignment statement to declare the type of a variable:
-
-.. code-block:: python
-
-   x = 1  # type: Union[int, str]
-
-We'll use both syntax variants in examples. The syntax variants are
-mostly interchangeable, but the variable annotation syntax allows
-defining the type of a variable without initialization, which is not
-possible with the comment syntax:
-
-.. code-block:: python
-
-   x: str  # Declare type of 'x' without initialization
-
 .. note::
 
    The best way to think about this is that the type annotation sets the
    type of the variable, not the type of the expression. To force the
-   type of an expression you can use :py:func:`cast(\<type\>, \<expression\>) <typing.cast>`.
+   type of an expression you can use :py:func:`cast(\<type\>, \<expression\>) <typing.c#ast>`.
```

Comment:
```suggestion
   type of an expression you can use :py:func:`cast(\<type\>, \<expression\>) <typing.cast>`.
```

Reply:
Sloppy, not sure what happened there, sorry about that